tests/frameworks/test_huggingface.py

```diff
@@ -178,8 +190,27 @@ def test_huggingface_log_model(tmp_dir, model, data, mocker, log_model, best):
         )
 
 
-def test_huggingface_pass_logger():
+@pytest.mark.parametrize("callback", [ExternalCallback, InternalCallback])
+def test_huggingface_pass_logger(callback):
     logger = Live("train_logs")
 
-    assert DVCLiveCallback().live is not logger
-    assert DVCLiveCallback(live=logger).live is logger
+    assert callback().live is not logger
+    assert callback(live=logger).live is logger
+
+
+@pytest.mark.parametrize("report_to", ["all", "dvclive", "none"])
+def test_huggingface_report_to(model, report_to):
```

Comment:
Since there are no real tests for the callbacks in `transformers`, added a test here to ensure `report_to` adds the dvclive callback when expected